packages/providers/upload-cloudinary/lib/index.js

```diff
@@ -10,7 +10,8 @@ const intoStream = require('into-stream');
 const { PayloadTooLargeError } = require('@strapi/utils').errors;
 
 module.exports = {
-  init(config) {
+  // eslint-disable-next-line no-unused-vars
+  init({ sizeLimit, ...config }) {
```

Comment:
If we remove the sizeLimit from the providerOptions, we should actually remove this.

Even if people are using it inside providerOptions, cloudinary and aws-s3 are not using this argument, so it wouldn't be breaking either.